src/client-bridge/src/api/client.py

```
@@ -0,0 +1,189 @@
+
+        Also stores the token's expiration so we can fail fast with a
+        clear error instead of waiting for a 401 from the backend.
+        """
+        clone = object.__new__(MessagesAPIClient)
+        clone.base_url = self.base_url
+        clone._client = self._client  # noqa: SLF001
+        clone._token = token
+        # Extract exp from the JWT (decode without verification — we issued it)
+        try:
+            payload = jwt.decode(token, options={"verify_signature": False})
+            clone._token_exp = payload.get("exp")
```

Ruff (0.15.4)

[error] Private member accessed: `_token_exp`

(SLF001)